examples/docker/example-package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "private": true,
+  "name": "docker-theia-trace-example",
+  "version": "0.1.0",
+  "theia": {
+    "target": "browser",
+    "frontend": {
+      "config": {
+        "applicationName": "Theia-Trace Example Application",
+        "preferences": {
+          "editor.autoSave": "on",
+          "trace-viewer.port": 8080
+        }
+      }
+    }
+  },
+  "dependencies": {
+    "@theia/core": "1.34.1",
```

Comment:
The version of the `theia-trace-extension` to 1.34.2 was updated in a recent commit. The version needs to be updated here as well. Otherwise the application can fail to start with the error reported [here](https://community.theia-ide.org/t/cannot-apply-injectable-decorator-multiple-times/2628). I got that error after fetching the PR and rebasing it. Could you please update the example-package.json for Theia 1.34.2 and rebase your code to the latest in master and update the PR? I tired it locally and it worked fine.

Reply:
Yes, this is an issue due to the fact that the theia-traceviewer npm package has no versioning, hence this file has always to be maintained to the "latest and greatest". The main goal of this commit is just to show how we can create a docker image of the theia-trace-extension project.

If we want to avoid issues as the one you report above, we should:
- include the building and testing of the docker image (including starting it) in the CI machinery
- start versioning theia-trace-extension and the theia-traceviewer npm package
- just add in the readme a description of how to handle the issue you reported above

Initially I had a script to copy the package.json from the browser folder, and create the example-package.json, but I thought it was ugly (also if that would solve the issue of "keeping up to date").

Let me know if I can do something more to improve this PR, otherwise I think it is good enough to have an example and it will be up to us to keep it up to date when we uplift.

Reply:
Thanks for the update. All good. We'll also update the Theia version in the example-packages.json when upgrading in the future.

Reply:
Regarding this, do you have any advice on how I could make a stable app (set to a particular theia-trace-extension version) so that it doesn't crash every time theia-trace-extension is updated? Would putting _latest_ instead of a specific version do the job or is the only solution to write the script that @frallax mentioned?

Reply:
The thing is, `theia-trace-extension` itself (published as `theia-traceviewer`) has no versioning, at all. Hence the necessary use of `next`, here. Now, the core theia libraries used internally across this whole `theia-trace-extension` repo (Docker parts included) are all pinned to `1.34.2` currently, which should be consistent enough. Goal being, that we keep maintaining all of these versions accordingly over time. We cannot use latest or next, because we need to pin a version of those libraries that is known to work as latest with this current repo codebase.

This should suffice, but anyone remains [welcome to contribute such maintenance acts](https://github.com/eclipse-cdt-cloud/theia-trace-extension/blob/master/CONTRIBUTING.md#uplifting-the-theia-dependencies-version), over time. While writing this, I just noticed that we [should mention docker now under here](https://github.com/eclipse-cdt-cloud/theia-trace-extension/blob/master/CONTRIBUTING.md#uplifting-the-theia-dependencies-version). So I'll push a PR to fix this soon.

Reply:
<=> PR #954 that I just pushed for review (a simple one).